### PR TITLE
Adding direct logout test.

### PR DESCRIPTION
**Pull request by charvimehradu**
Pull Request for Issue # 

Adding direct logout test.

### CODE REVIEW

File: tests/System/integration/site/components/com_users/Logout.cy.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+describe('Test in frontend that the users logout view ', () => {
```
Comment (laoneo) [inline tests/System/integration/site/components/com_users/Logout.cy.js]:
```suggestion
describe('Test in frontend that the users logout view', () => {
```

File: tests/System/integration/site/components/com_users/Logout.cy.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    cy.get('#system-message-container').should('contain.text', 'You have been logged out.');
+  });
+
+  it('can log out the user in a menu item', () => {
+    cy.doFrontendLogin();
+    cy.db_createMenuItem({ title: 'Automated logout', link: 'index.php?option=com_users&view=login&layout=logout&task=user.menulogout' });
+    cy.visit('/');
+    cy.get('a:contains(Automated logout)').click();
+
+    cy.get('#system-message-container').should('contain.text', 'You have been logged out.');
```
Comment (laoneo) [inline tests/System/integration/site/components/com_users/Logout.cy.js]: This code block needs to go into a .then block like we do in the other tests

File: tests/System/integration/site/components/com_users/Logout.cy.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+describe('Test in frontend that the users logout view ', () => {
+  it('can log out the user without a menu item', () => {
+    cy.doFrontendLogin();
+    cy.visit('index.php?option=com_users&view=login&layout=logout&task=user.menulogout');
+
+    cy.get('#system-message-container').should('contain.text', 'You have been logged out.');
+  });
+
+  it('can log out the user in a menu item', () => {
+    cy.doFrontendLogin();
```
Comment (laoneo) [inline tests/System/integration/site/components/com_users/Logout.cy.js]: This needs to go after the createMenuItem call also into the .then block

**Comment by laoneo** [comment]
The tests are failing

**Comment by brianteeman** [comment]
> The tests are failing

that may well be correct see #37763

**Comment by charvimehradu** [comment]
I'm not sure why these aren't working over here, but they do on my system.

LogoutTestPass

**Comment by laoneo** [comment]
In the artefacts part you will find some information why the test fails https://ci.joomla.org/joomla/joomla-cms/64752/1/37

File: tests/System/integration/site/components/com_users/Logout.cy.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
+describe('Test in frontend that the users logout view', () => {
+  it('can log out the user without a menu item', () => {
+    // Make sure we are really logged in
+    cy.doFrontendLogin(null, null, false);
+    cy.visit('/index.php?option=com_users&view=login&layout=logout&task=user.menulogout');
```
Comment (laoneo) [inline tests/System/integration/site/components/com_users/Logout.cy.js:5]: Just for information the slash is needed as cypress has an issue when a query value contains a dot

**Comment by laoneo** [comment]
Thanks!

**Pull request merged**